packages/lib-utils/src/app/redux/reducers/k8s/selector.ts

```diff
@@ -0,0 +1,6 @@
+import type { K8sState, SDKStoreState } from '../../../../types/redux';
+
+export const getReduxIdPayload = (state: SDKStoreState, reduxId: string) =>
+  state?.k8s?.get(reduxId);
```

Comment:
```suggestion
  state.k8s.get(reduxId);
```
They are not optional in the call signature, they can't be chained as such.

Reply:
This looks to be a problem -- maybe we leave it as-is 🤔

Reply:
I think since `SDKStoreState` can yield `undefined` (see https://github.com/openshift/dynamic-plugin-sdk/pull/37/files#r819902820), this is required.